CMakeLists.txt

```diff
@@ -35,8 +40,12 @@ set(XPROPERTY_HEADERS
     ${XPROPERTY_INCLUDE_DIR}/xproperty/xproperty_config.hpp
 )
 
+add_library(xproperty INTERFACE)
+target_include_directories(xproperty INTERFACE $<BUILD_INTERFACE:${XTENSOR_INCLUDE_DIR}>
```

Comment:
There's a typo here, it should be ${XPROPERTY_INCLUDE_DIR}

Reply:
Fixed.